code/style.css

```diff
@@ -64,10 +72,10 @@ select {
   width: 135px;
   height: 165px;
   overflow: hidden;
-  border: 3px solid var(--secondary);
-  border-radius: 4px;
-  margin: 3px;
-  display: flex;
+  background-color: white;
+  border-radius: 70px;
```

Comment:
really nice with the rounded corners, and the color scheme looked really well put together